vm/nova/vm_test.go

```diff
@@ -6177,23 +6879,24 @@ func TestManaRewardsClaimingStaking(t *testing.T) {
 	transaction := &iotago.Transaction{
 		API: testAPI,
 		TransactionEssence: &iotago.TransactionEssence{
-			Inputs: inputIDs.UTXOInputs(),
+			Inputs:       inputIDs.UTXOInputs(),
+			CreationSlot: currentSlot,
 		},
 		Outputs: iotago.TxEssenceOutputs{
 			&iotago.AccountOutput{
-				Amount:         OneMi * 5,
-				AccountID:      accountIdent.AccountID(),
-				StateIndex:     2,
-				StateMetadata:  nil,
+				Amount:         OneIOTA * 5,
+				Mana:           12043,
```

Comment:
Where is this number coming from?

Reply:
From executing this test :D

Reply:
Can you document this :)?

Reply:
Changed it to being calculated automatically.